metriq_gym/poll_jobs.py

```diff
@@ -0,0 +1,42 @@
+"""Quantum volume + CLOPS + EPLG in Qiskit."""
+import json
+import math
+import statistics
+import logging
+
+from scipy.stats import binom
+
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit.providers import JobStatus
+
+from metriq_gym.process import poll_job_results, calc_stats
+from metriq_gym.parse import parse_arguments
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+def main():
+    args = parse_arguments()
+
+    if args.token:
+        QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, set_as_default=True, overwrite=True)
+
+    logging.info(f"Polling for job results.")
```

Comment:
Technically, any of the strings that don't have argument-based outputs don't need to be `f`-strings.